bundle/jsky.app.ot/src/main/java/jsky/app/ot/progadmin/TimeAcctEditor.java

```diff
@@ -87,9 +87,9 @@ public TimeAcctAllocation getAllocation() {
 
     private void showTotalTime(final TimeAcctAllocation alloc) {
         final TimeAcctAward a = alloc.getSum();
-        ui.getTotalAwardLabel()  .setText(nf.format(a.getTotalHours()));
-        ui.getProgramAwardLabel().setText(nf.format(a.getProgramHours()));
-        ui.getPartnerAwardLabel().setText(nf.format(a.getPartnerHours()));
+        ui.getTotalAwardLabel()  .setText(nf_dec.format(a.getTotalHours()));
+        ui.getProgramAwardLabel().setText(nf_dec.format(a.getProgramHours()));
+        ui.getPartnerAwardLabel().setText(nf_dec.format(a.getPartnerHours()));
```

Comment:
I'll ask Bryan for clarification. My understanding of the JIRA task was that he wanted us to deliberately show them to emphasize the precision differences.